mitxpro/settings.py

```diff
@@ -612,7 +613,12 @@
             600,
             description="How many seconds between Hubspot API error checks",
         ),
-    }
+    },
+    # a job is scheduled to run at every-day-at-midnight
+    "generate-course-certificate-daily": {
+        "task": "courses.tasks.generate_course_certificates",
+        "schedule": crontab(0, 0, "*", "*", "*"),
```

Comment:
Optional enhancement: I think we could make this configurable if we add 2 settings.

```python
CRON_COURSE_CERTIFICATES_HOURS = get_string(
  "CRON_COURSE_CERTIFICATES_HOURS", 
  0,
  "'hours' value for the `generate-course-certificate` scheduled task (defaults to midnight)"
)
CRON_COURSE_CERTIFICATES_DAYS = get_string(
  "CRON_COURSE_CERTIFICATES_DAYS", 
  None,
  "'day_of_week' value for the `generate-course-certificate` scheduled task (default is None, meaning it will run once a day)"
)
# ...
CELERY_BEAT_SCHEDULE = {
  # ...
  "generate-course-certificate-daily": {
    # ...
    "schedule": crontab(
      hours=CRON_COURSE_CERTIFICATES_HOURS,
      minute=0,
      day_of_week=CRON_COURSE_CERTIFICATES_DAYS or '*',
      day_of_month='*', 
      month_of_year='*'
    )
```